README.md

```diff
@@ -21,7 +21,7 @@ If only 1 parameter is supplied this will run directly on salt
 
 ## Examples
 
-`csalt "group1 gp:group2" test.ping`
+`csalt "group1 group2:gp" test.ping`
```

Comment:
For another PR perhaps but it would be nice if the help output had an example like this